<launch>
	<node pkg="x52_joyext" type="x52_menu_creator_node" name="x52_menu_creator" output="screen"> 


			<param name="menu_setup" value="test1;2;0;/narf;0|test2;1;1;/gnampf;1"/>
			<!-- Remap Topics 
			<remap from="~/in" to="/joy" />
			<remap from="~/mfd_text" to="/x52/mfd_text" />-->	
	</node>
</launch>